cmd/create/addon/cmd.go

```diff
@@ -130,12 +131,15 @@ func run(_ *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
-	reporter.Debugf("Installing add-on '%s' on cluster '%s'", addOnID, clusterKey)
-	err = clusterprovider.InstallAddOn(clustersCollection, clusterKey, awsCreator.ARN, addOnID)
-	if err != nil {
-		reporter.Errorf("Failed to add add-on installation '%s' for cluster '%s': %s", addOnID, clusterKey, err)
-		os.Exit(1)
+	reporter.Warnf("Once installed, add-ons cannot be uninstalled")
+	if confirm.Confirm("install add-on '%s' on cluster '%s'", addOnID, clusterKey) {
```

Comment:
Should we add a global `--force` or `--confirm` flag that ignores these for users who might script this?

Reply:
Good idea, working on it now

Reply:
done